app/controllers/onboarding/telegram_controller.rb

```diff
@@ -42,11 +42,13 @@ def update
 
       return render 'onboarding/unauthorized', status: :unauthorized unless @contributor
 
-      if @contributor&.update(
+      @contributor.assign_attributes(
```

Comment:
There’s no need for null-safe navigation in this case, as we’re returning in case `@contributor` isn’t set in the line before.